### ISSUE TITLE
Sample script running the sample data

### ISSUE THREAD

**Issue by usptact** [abhyudaynj/LSTM-CRF-models#1]
Thanks for sharing the code! Could you please provide the commands that use sample data for prediction and model training? It is not very clear to me what arguments to specify to the scripts.

In my case,  have a collection of annotated short texts. I would like to run your code and build a model from my data.

Thanks again!

**Comment by abhyudaynj** [comment]
Hello, A sample parameter settings for both training and prediction are provided as default arguments. So when you execute the training or deployement statements without arguments, it falls back to a default setting which can be seen by running both scripts in help (-h ) mode. 

I would recommend just running the sample statements provided in the README.md file, and then trying to change the learning rate, network size and batch size parameters first. This should give you a reasonable idea on how the model works on your problem. Let me know if there are any issues.
